app/models/waiting_list.rb

```diff
@@ -6,7 +6,7 @@ class WaitingList < ActiveRecord::Base
   def remove(entry)
     update_column :entries, entries - [entry.id]
   end
-
+  
```

Comment:
Meaningless blank space, surprised that RuboCop doesn't beat you over the head with it